llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -8884,8 +8884,9 @@ bool SIInstrInfo::isBasicBlockPrologue(const MachineInstr &MI,
   // FIXME: Copies inserted in the block prolog for live-range split should also
```

Comment:
This FIXME isn't relevant anymore. We are including only the SGPR spills to the prolog.